### DISCUSSION TITLE
Custom button, add option

### DISCUSSION THREAD

**Discussion by jijoknotengo** [osmandapp/OsmAnd#20942] [Ideas]
On the map options tabs, there is a hide button with different options, hiding POIs on big cities really help the smooth navigation. Adding that option to the quick action button would do it faster.

I know there is the option to hide different POIs, but that one makes all pois orange ( also crashed the app )

That's my little idea, cheers

**Comment by DmitryAlexei** [comment]
In the latest nightly version, you can create a quick action button that will show / hide specified POIs (but not all).
As a workaround, it is possible to create a separate profile for navigation with POI icons and names disabled.
